subsys/bluetooth/mesh/Kconfig

```diff
@@ -363,6 +363,9 @@ config BT_MESH_ADV_LEGACY
 	  advertising is significantly slower than the extended advertising, but
 	  is supported by all controllers.
 
+	  WARNING: The legacy advertiser can occasionally do more message
+	  retransmissions than requested.
+
```

Comment:
```suggestion
	  WARNING: The legacy advertiser can occasionally do more message
	  retransmissions than requested because of limitations of HCI 
	  interface API.
	  
```